### ISSUE TITLE
[BUG] wwctl overlay build with flags (-a|--all) no longer works

### ISSUE THREAD

**Issue by tcooper** [hpcng/warewulf#134]
### Describe Bug
Recent changes to overlay commands break some pre-existing behavior.

For example, the `-a` flag for `wwctl overlay build...` command no longer works.

### To Reproduce
1. Modify runtime overlay
2. Attempt to build all overlays
3. See error

#### Help text
```
[vagrant@mgr ~]$ sudo wwctl overlay build --help
This command will build a system or runtime overlay.

Usage:
  wwctl overlay build [flags] <overlay kind> <overlay name>

Flags:
  -a, --all    Build all overlays (runtime and system)
  -h, --help   help for build

Global Flags:
  -d, --debug     Run with debugging messages enabled.
  -v, --verbose   Run with increased verbosity.
```

#### Previous (working) execution
```
[vagrant@mgr ~]$ sudo wwctl overlay build -a
ERROR: accepts 2 arg(s), received 0
```

#### Attempt to build default system and runtime overlays
```
[vagrant@mgr ~]$ wwctl overlay build -a default
ERROR: accepts 2 arg(s), received 1
```

#### Working commands
```
[vagrant@mgr ~]$ sudo wwctl overlay build runtime default
[INFO]     Updating Runtime Overlays...
[INFO]     cn0                                : Done
[INFO]     cn1                                : Done
[vagrant@mgr ~]$ sudo wwctl overlay build system default
[INFO]     Updating System Overlays...
[INFO]     cn0                                : Done
[INFO]     cn1                                : Done
```

### Expected behavior
Do one of...
- re-implement the behavior currently provided by the `-a|--all` option to `wwctl overlay build`
- remove the `-a|--all` option from the `wwctl overlay build ...` CLI option (and help) completely,

### System (please complete the following information):
 - OS: Rocky Linux 8.4
 - Warewulf Version: v4.1.0 (git hash acf299093609303efd8c56659a6b8aa0ffa10369)

**Comment by gmkurtzer** [comment]
Hi @tcooper,

Fixes implemented in PR #138, thanks for the report!

Greg

**Comment by gmkurtzer** [comment]
@tcooper Just wanted to let you know I'm closing this as I believe it is working properly now. Please feel free to reopen if needed. Thanks!
